INSTALL.md

```diff
@@ -106,7 +106,7 @@ Required build tools:
 Required build dependencies:
 
 * Qt >= 5.15.2
-* Proj >= 7.2.0
+* Proj >= 8.2.0
```

Comment:
8.1 or 8.2 (cf check in CMakeListst.txt) ?

Reply:
Fixed